YANG/ccamp/layer1-types/ietf-layer1-types.yang

```diff
@@ -1145,40 +1165,48 @@ module ietf-layer1-types {
 
        This grouping should be used together with the
        otn-label-range-info and otn-label-start-end groupings to
-       provide OTN technology-specific label information to the models
-       which use the label-restriction-info grouping defined in the
-       module ietf-te-types.";
-    choice range-type {
+       provide OTN technology-specific label information to the
+       models which use the label-restriction-info grouping defined
+       in the module ietf-te-types.";
+    container otn {
       description
-        "OTN label range type, either TPN range or TS range";
-      case trib-port {
-        leaf otn-tpn {
-          when "../../range-type = 'trib-port'" {
-            description 
-              "Valid only when range-type represented by trib-port";
-          } 
-          type otn-tpn;
-          description
-            "Label step which represents possible increments for 
-             Tributary Port Number.";
-          reference
-            "RFC7139: GMPLS Signaling Extensions for Control of 
-             Evolving G.709 Optical Transport Networks.";
+        "Label step for OTN";
+      choice range-type {
+        description
+          "OTN label range type, either TPN range or TS range";
+        case trib-port {
+          leaf otn-tpn {
+            when "../../../otn-label-range/range-type = 
+                  'trib-port'" {
+              description 
+                "Valid only when range-type represented by
+                trib-port";
+            } 
+            type otn-tpn;
+            description
+              "Label step which represents possible increments for 
+              Tributary Port Number.";
+            reference
+              "RFC7139: GMPLS Signaling Extensions for Control of 
+              Evolving G.709 Optical Transport Networks.";
+          }
         }
-      }
-      case trib-slot {
-        leaf otn-ts {
-          when "../../range-type = 'trib-slot'" {
-            description 
-              "Valid only when range-type represented by trib-slot";
-          } 
-          type otn-ts; 
-          description
-            "Label step which represents possible increments for 
-             Tributary Slot Number.";
-          reference
-            "RFC7139: GMPLS Signaling Extensions for Control of
-             Evolving G.709 Optical Transport Networks.";
+        case trib-slot {
+          leaf otn-ts {
```

Comment:
If we add an "otn" container then here "otn-" prefix is really unnecessary. Suggest changing "otn-ts" to just "trib-slot"

Reply:
As soon as there is a description e.g. tpn ...is tributary port number I do not think a longer name is needed.

Reply:
Addressed in https://github.com/haomianzheng/IETF-ACTN-YANG-Model/pull/104/commits/b80b3a8f5fbf26efc4b237f06b859a03e133b297